### ISSUE TITLE
Rewording of RGI box on front page

### ISSUE THREAD

**Issue by anderspeders** [NRGI/resourcedata.org#124]
## Header

From: 2017 RESOURCE GOVERNANCE INDEX

To: 2017 RGI SOURCE LIBRARY 

## Text
Change from: 
The Resource Governance Index measures the quality of resource governance in 81 countries. Explore the nearly 10,000 source documents used for the country assessments.

To: 
The Resource Governance Index (RGI) Source Library contains almost 10,000 source documents which were used to verify the information from the 149 conducted survey questions across 89 assessments. Each document can be searched by question, assessment type, country and subcomponent.

**Comment by mattfullerton** [comment]
I would reword the last sentence?

"The documents can be ...."  instead of "Each document"

**Comment by anderspeders** [comment]
Yes, please proceed.

**Comment by anderspeders** [comment]
Could you change the front page box link:

from: https://www.resourcedata.org/organization/edit/rgi

to: https://www.resourcedata.org/organization/resource-governance-index

**Comment by anderspeders** [comment]
@iprunache Would you be able to conduct this change tomorrow Tuesday AM?

**Comment by iprunache** [comment]
@anderspeders can we close this?

**Comment by anderspeders** [comment]
thanks

**Comment by mattfullerton** [comment]
This has been done. We should transition the import scripts to use this organization name.
